apps/communications/contacts/js/contacts.js

```diff
@@ -692,11 +699,37 @@ var Contacts = (function() {
     window.removeEventListener('DOMContentLoaded', onLoad);
   });
 
+  sessionStorage.setItem('oncontactchange', null);
```

Comment:
Was thinking that maybe we should change the key. Right now is associated to the method oncontactchange from the api, but it could be more readable if we could name it something like 'changes' or something similar.